lib/fog/vsphere/requests/compute/list_clusters.rb

```diff
@@ -41,7 +41,7 @@ def cluster_attributes cluster, datacenter_name
 
         def cluster_path(cluster, datacenter_name)
           datacenter = find_raw_datacenter(datacenter_name)
-          cluster.pretty_path.gsub(/(#{datacenter.name}|#{datacenter.hostFolder.name})\//,'')
+          cluster.pretty_path.gsub(/(#{datacenter_name}|#{datacenter.hostFolder.name})\//,'')
         end
       end
```

Comment:
I think not. IMHO this variable was introduced only to help set the full_path attribute of cluster (just because of the folder structure). You are right that it should be named a bit better.